Ft html

BarnaTB opened the pull request:
Hey please review my code before I merge into develop and leave some feedback @louiCoder @dnuwa @judeinno . Thanks

BarnaTB committed:
Create branch for UI files [#159639299]

BarnaTB committed:
Add HTML files

BarnaTB committed:
Modify README installation guidelines

BarnaTB committed:
Add html page structure [Fixes #159638576]

BarnaTB committed:
Add html page structure [Fixes #159638603]

BarnaTB committed:
Add home html page structure [Fixes #159638752]

BarnaTB committed:
Add home page css styling [Fixes #159638752]

BarnaTB committed:
ft(User home page): This commit fixes the responsive design for
the home page navigation bar.
[Fixes #159638752]

BarnaTB committed:
Add css styling for sign up page [Fixes #159638576]

BarnaTB committed:
Move index.html file

BarnaTB committed:
Add link to Andela in acknowledgements

BarnaTB committed:
Commit to finish the design of the User home page [Delivers #159638752]

BarnaTB committed:
Fix margin of answer buttons inside question div [Fixes #159638752]

BarnaTB committed:
Commit to deliver the feature for a user to post a question [Delivers #159638839]

BarnaTB committed:
Modify mobile media query button design [Fixes #159638752]

BarnaTB committed:
Add icons folder to gitignore file

BarnaTB committed:
ft(user profile page)Add user page: Commit delivers the feature for
a user profile page where the user can view recently asked
 questions, view question with highest number of answers, number
of questions asked and number of answers given.
[Delivers #159639271 #159639196 #159638953]

BarnaTB committed:
ft(User can view all answers to question):Add html and css styling.
Commit delivers feature for user to view all answers to a question
[Delivers #159638752 #159638801]

BarnaTB committed:
ft(User can view all answers to any question): Add html and css
styling.
Commit delivers a feature that enables the user to view the
answers to all answers to a question
[Delivers #159638801]

BarnaTB committed:
Remove commented code

BarnaTB committed:
Change link for answer button

BarnaTB committed:
Change div class name

BarnaTB committed:
Fix merge conflicts

BarnaTB committed:
Fix merge conflicts

BarnaTB committed:
Add contact page

BarnaTB committed:
Remove About Us from nav-bar

BarnaTB committed:
Fix merge conflicts

BarnaTB committed:
Add Icons

BarnaTB committed:
Merge branch 'ft-HTML' into gh-pages
Add icons from ft-HTML branch

BarnaTB committed:
Restructure JS file

BarnaTB committed:
Add github pages link

BarnaTB committed:
Add github pages link

BarnaTB committed:
Add link to github pages

BarnaTB committed:
Change JS folder name to scripts

BarnaTB committed:
Fix merge conflicts in README.md and rename JS folder

dnuwa commented:
looking at your UI, I like the way you have worked with the colors and the page structure good. though if you could add some borders it would also be great. In general, you have put up good work.

BarnaTB committed:
bg(User cannot view other navigation options)Fix wrong link to JS
file. Commit to deliver a bug fix where user could not see other
navigation options in mobile view and change background color.
[Fixes #159873839]

BarnaTB committed:
ft(User profile page)Add css styling to the user profile page.
Commit to fix the styling on the user profile page.
[Fix #159639271 #159639196 #159638953]
